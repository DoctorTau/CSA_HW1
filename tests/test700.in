401
-397 7385 -9213 1944 223 608 -422 -9582 5896 -4743 6304 5095 -9283 6278 -9233 974 7862 -3788 -7355 9632 1270 1413 -3321 2241 -6388 -2600 -3439 -774 -5772 -4270 8845 4026 -8514 -2304 4502 -1654 6135 -9065 -3794 6438 -4936 1009 -7628 3202 -1389 -9749 -6742 5789 -9409 8734 3284 -5563 -9405 -1149 2799 -9110 -9863 6509 8022 5375 -9010 5756 -4699 7114 -4860 -3415 897 -2447 -4813 5196 4178 3286 699 9485 -5632 321 -9554 -5487 9179 8679 2879 -2028 -2804 -8093 -3766 629 3118 -791 4709 -2481 2114 -6599 3473 4805 -9559 5421 7805 25 -7782 2899 2163 -1429 -7901 -6906 4110 -6394 -4116 1594 -4102 275 -38 -6007 -7071 3430 3103 -8895 1718 3340 -4689 -5549 451 5790 -1860 -8503 3992 8436 6776 -55 4580 -9698 -9370 4863 -8564 -912 -7725 -2692 4286 -5617 -9071 -1026 2940 -2991 -5471 -9981 3404 3991 7395 -987 9507 5238 7855 -4873 -6223 -4765 2976 8557 9505 3640 -1138 -1482 -5276 3517 3469 -4199 692 -7521 -8747 6352 4991 1150 4715 7575 1477 5972 -2433 -4617 8644 -1464 2658 -7967 6055 -8678 7114 -6177 -2414 6889 9499 -700 -9596 3616 2749 -7511 -6249 -8144 -1920 -5393 1927 1911 219 1787 7303 6748 9448 4743 773 -826 -9178 -3835 8271 6477 1461 1954 5887 -336 4266 -7394 -8220 129 -8180 731 -7770 -3463 1815 -2137 2035 7412 -7708 5618 4033 4183 -8861 -1250 3473 -1259 9811 7399 -1050 -9701 -3968 4272 -2245 -6347 -6696 -6567 -4104 -9016 4090 -1235 7652 -103 3661 -3031 -1271 6311 -962 6108 2151 7604 -3061 2830 9758 7106 2758 1130 581 -6957 2894 -9868 -1675 1945 -6189 -7988 5271 -1275 3024 8420 -1163 211 -4820 3127 -2425 -4444 7272 -2467 -6333 6436 7271 -5391 1007 1862 7053 -7078 -375 7575 7347 9817 1041 -3663 -3341 -7414 1605 -2369 3587 -6655 -2993 37 -4981 -768 -8372 2272 300 1057 -1592 -8768 699 616 3431 1177 -7106 9877 -9236 114 -1843 -1835 -1928 7424 5204 -3852 -8808 -9233 -5570 3521 -5998 9449 2362 -2115 6932 -8217 7840 -8794 7871 -5064 -7966 -4517 -4168 -3495 5542 7048 -1845 7304 5014 -6244 -8950 8141 -3428 -5783 -7544 3710 -6912 -8844 -1808 -8369 -6724 -3423 -725 -2893 1883 4579 -5628 9451 9247 4981 -833 7737 673 -1770 722 849 2979 6112 -3392 -8035 2201 9181 -537 -5617 2019 -1600 -3687 4743 1755 -9366 4378 -1717 6231 -8367 -104 -5493 7058 -443 6009 