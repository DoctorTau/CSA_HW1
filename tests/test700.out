-422 -9582 -3788 -6388 -2600 -774 -5772 -4270 -8514 -2304 -1654 -3794 -4936 -7628 -6742 -9110 -9010 -4860 -5632 -9554 -2028 -2804 -3766 -7782 -6906 -6394 -4116 -4102 -38 -1860 -9698 -9370 -8564 -912 -2692 -1026 -1138 -1482 -5276 -1464 -8678 -2414 -700 -9596 -8144 -1920 -826 -9178 -336 -7394 -8220 -8180 -7770 -7708 -1250 -1050 -3968 -6696 -4104 -9016 -962 -9868 -7988 -4820 -4444 -7078 -7414 -768 -8372 -1592 -8768 -7106 -9236 -1928 -3852 -8808 -5570 -5998 -8794 -5064 -7966 -4168 -6244 -8950 -3428 -7544 -6912 -8844 -1808 -6724 -5628 -1770 -3392 -1600 -9366 -104 